code/modules/research/designs/misc_designs.dm

```diff
@@ -67,6 +67,17 @@
 //////////////////Misc///////////////////
 /////////////////////////////////////////
 
+
+/datum/design/plasticducky
+	name = "Rubber Ducky"
+	desc = "The classic Nanotracen design for competitively priced bath based duck toys. No need for fancy Waffle co. rubber, buy Plastic Ducks today!"
+	id = "plasticducky"
+	build_type = PROTOLATHE
+	materials = list(/datum/material/plastic = 1000)

```

Comment:
Someone tell me if this is sane or not.

Reply:
it's half a sheet and code is fine.

Reply:
This text appears nowhere in-game if it makes you happy

Reply:
>nanotracen